index.html

```diff
@@ -195,7 +198,8 @@ <h3 class="contact-header">Contact me</h3>
           tabindex="36"
           aria-label="Please input your comment here">
```

Comment:
- [OPTIONAL] `required` attribute is missing from your textarea tag. Kindly add it to complete total validation for your form.